src/Sarif/Baseline/ResultMatching/SarifLogMatcher.cs

```diff
@@ -180,7 +181,7 @@ private Rule GetRuleFromResources(Result result, IDictionary<string, Rule> rules
 
         private SarifLog ConstructSarifLogFromMatchedResults(IEnumerable<MatchedResults> results, IEnumerable<Run> previous, IEnumerable<Run> currentRuns)
```

Comment:
>previous [](start = 115, length = 8)

While we're in here, can we rename this to `previousRuns` for consistency with `currentRuns`?